Automattic/jetpack
# Plan: Add support for Monthly pay period to products.

Make the monthly and yearly toggle work for Products. 

Fixes https://github.com/Automattic/jetpack/issues/15094

This is what the page looks like now. 

<img width="1143" alt="Screen Shot 2020-03-31 at 5 26 08 PM" src="https://user-images.githubusercontent.com/115071/78044292-c105ba00-7374-11ea-8e40-89cf61ba31fa.png">


#### Changes proposed in this Pull Request:
* This PR adds a plan duration to the redux store. This way the pay period can be accessed from any component. 


#### Testing instructions:
1. Build this PR. 
2. Notice that you can toggle the monthly and yearly toggle on the plans page in wp-admin.
3. Notice that you can buy the correct product. 

#### Proposed changelog entry for your changes:
Make the monthly prices available in wp-admin plans page.
